web/source/keymanweb.js

```diff
@@ -2620,13 +2607,7 @@ if(!window['keyman']['initialized']) {
       ////keymanweb._SelectionControl = null;    
       var keyboardID = keymanweb._ActiveKeyboard ? keymanweb._ActiveKeyboard['KI'] : '';
```

Comment:
`keyboardID` can be removed because it's used internally in `keymanweb._BlurKeyboardSettings()`